### DISCUSSION TITLE
How to work with static files?

### DISCUSSION THREAD

**Discussion by ochronus** [codingjoe/django-pictures#101] [Q&A]
Might be a silly question (as in out of the context) but is there a canonical way to work with static files? I mean files manually placed during development, and then served using the staticfiles app.

**Comment by codingjoe** [comment]
Hi @ochronus,

Thank you for reaching out. Rest assured, there are no silly questions. However, I don't know I fully understand yours. Would you care to expand on your use case a little?

Cheers!
Joe
